crates/forge/bin/cmd/test/mod.rs

```diff
@@ -397,13 +397,6 @@ impl TestOutcome {
             }
             println!();
         }
-
-        let successes = self.successes().count();
-        println!(
-            "Encountered a total of {} failing tests, {} tests succeeded",
-            Paint::red(failures.to_string()),
-            Paint::green(successes.to_string())
-        );
```

Comment:
hmm why remove this? don't think we should

Reply:
We already display the aggregated test summary for every runs, which was added in https://github.com/foundry-rs/foundry/pull/5266.

```bash
Ran 4 test suites: 52 tests passed, 2 failed, 0 skipped (54 total tests)
Failing tests:
....

Encountered a total of 2 failing tests, 52 tests succeeded
```

The output is redundant in-case of test failures, as the total failed and passed test counts are displayed once again at the end.  See attached screenshots above for reference.

Reply:
Right, I understand your point—but I lean on keeping this for the sake of not making a breaking change on the tool's output. Let's keep it for now as it's something ppl expect to be there.

Reply:
Sounds good